crates/esthri-test/data/.gitattributes

```diff
@@ -0,0 +1 @@
+crates/esthri-test/data/delete-me.txt filter=lfs diff=lfs merge=lfs -text
```

Comment:
I though you said this file wasn't supposed to be tracked? 

```
git rm crates/esthri-test/data/delete-me.txt     
git commit -m ....
```

to remove it if so (also should remove this gitattributes file

Reply:
A mistake produced via `git add .`. resolved.